### ISSUE TITLE
Normalize slashes cross-platform

### ISSUE THREAD

**Issue by dy** [ttag-org/ttag-cli#85]
We're doing `npx ttag update ./i18n/en-CA.po ./src` command.
In OSX that generates forward slashes:
```po
#: src/constants.js:10
```
In windows that generates back slashes:
```po
#: src\constants.js:10
```
That creates useless noise in git diff.
Can that be prob normalized?

**Comment by AlexMost** [comment]
Hi @dy!
Is it ok if we will leave only the first way of formatting?

**Comment by dy** [comment]
@AlexMost absolutely, I prefer `/` - that's the standard path style and windows understands that too.

**Comment by joaogardenberg** [comment]
Any hints on how to do that?
